docs/source/support/release-notes.rst

```diff
@@ -78,10 +85,6 @@ Java Bindings
 
 - **SDK**: The Windows installer no longer requires elevated privileges.
 
-- **DAML Assistant**: We've built a new and improved version of the SDK assistant, replacing ``da`` commands with ``daml`` commands.
```

Comment:
Let's put it back how it was:
```
- **DAML Assistant**: The assistant handles network failures gracefully.
```

Reply:
I removed this in my previous PR because I didn't think it made sense to have a release note for improvements to a tool that hadn't really been released to users yet.